views/album.ejs

```diff
@@ -30,8 +32,8 @@
     <% photos.forEach(function(photo){ %>
       <li class="thumb">
         <a href="/<%= photo.url %>">
-          <img src="/<%= photo.src %>?tn=1" alt="<%= photo.name%>"><br />
-          <%= photo.name%>
+          <div class="image"><img src="/<%= photo.src %>?tn=1" alt="<%= photo.name%>"><br /></div>
```

Comment:
I'm not sure what extra semantic meaning adding the "image" div adds which can't already be achieved in CSS? 
There's only ever going to be 1x `img` child of a `.thumb`?